packages/editor-ui/src/App.vue

```diff
@@ -181,6 +181,21 @@ export default mixins(newVersions, showMessage, userHelpers).extend({
 				window.document.body.classList.add(`theme-${theme}`);
 			}
 		},
+		checkForExecutionsCount() {
+			let acc = 0;
+			const interval = setInterval(async () => {
+				try {
+					const plan = await this.usersStore.getOwnerCurrentPLan();
+					if (plan.metadata.slug !== 'trial-1') {
```

Comment:
I would check the group instead here.. or else that `planId === 0`.. since we will probably introduce `trial-2` soon to test different trial lengths

Reply:
Changing it to check whether the slug includes 'trial'